test/rununittests.py

```diff
@@ -27,30 +27,31 @@ def run( self, options ):
 def runAllTests( exe_path, use_valgrind=False ):
     test_proxy = TestProxy( exe_path, use_valgrind=use_valgrind )
     status, test_names = test_proxy.run( ['--list-tests'] )
+    print(test_names)
     if not status:
-        print >> sys.stderr, "Failed to obtain unit tests list:\n" + test_names
+        print("Failed to obtain unit tests list:\n" + test_names, file=sys.stderr)

```

Comment:
Actually this is one thing that is really annoying to support in both python 2 and 3 -- using print() to write to stderr.  I don't see any good reason why this needs to go to stderr though -- we should probably just make this a normal print statement.